Doc/library/idle.rst

```diff
@@ -1,7 +1,9 @@
 .. _idle:
 
-IDLE
-====
+:mod:`idlelib` --- IDLE
+=======================
+
+.. module:: idlelib
 
 .. moduleauthor:: Guido van Rossum <guido@python.org>
```

Comment:
As long as we're making updates to this, while I originally wrote it, maybe the more useful information is the current maintainer (Terry Reedy).